### DISCUSSION TITLE
Specify the directory path in the command line

### DISCUSSION THREAD

**Discussion by jbbarre** [jbbarre/ISM_SimulationChecker#9] [General]
Hello, based on a suggestion of @pmathiot, I created a [branch](https://github.com/jbbarre/ISM_SimulationChecker/tree/argparse) to modify the code in the way we can specify the path directly in the command line instead of changing the variable in the code.  
The new way of launching the script would be : `python compliance_checker.py '/mnt/path/to/the/repo'`

Do I merge the branch and include this in the main ? @hseroussi, what is your opinion?

**Comment by hseroussi** [comment]
@pmathiot and @jbbarre : both options would work and I don't have a strong preference. The reason I thought we could keep the path in the compliance_checker.py is that people will mostly check the same directory many times so they should not have to change that too often
